api-server/test/events.e2e-spec.ts

```diff
@@ -107,7 +116,7 @@ describe('Events API', () => {
           const paginatedEventReturned = events[i];
           const expectedEvent = allEvents[i];
 
-          assertEventsEqual(paginatedEventReturned, expectedEvent);
+          assertEventsEqual(paginatedEventReturned, expectedEvent, false);
```

Comment:
this was a fun bug I caught. these tests should have been failing out setting that last flag to false, which tells the assertion whether it would expect the organizer e-mail to be there. We weren't filtering it properly on these endpoints, now we are